fl4health/utils/partitioners.py

```diff
@@ -0,0 +1,99 @@
+import math
+from logging import INFO
+from typing import List, Optional, Tuple, TypeVar, Union
+
+import numpy as np
+import torch
+from flwr.common.logger import log
+
+from fl4health.utils.dataset import DictionaryDataset, TensorDataset, select_by_indices
+
+T = TypeVar("T")
+D = TypeVar("D", bound=Union[TensorDataset, DictionaryDataset])
+
+
+class LabelBasedDirichletAllocation:
```

Comment:
In order to be consistent with DirichletLabelBasedSampler maybe we can rename this as DirichletLabelBasedAllocation?

Reply:
Will do!